File: src/logica/Desafio.js
const valorSalario = parseFloat(prompt("Qual o seu salário?"));
const valorBeneficio = parseFloat(prompt("Qual o valor do beneficio?"));

const valorImposto = calcularImposto(valorSalario);

const saida = (valorSalario - valorImposto) + valorBeneficio;

function calcularImposto(salario){
 let aliquota;
 if (valorSalario >= 0 && valorSalario <= 1100) {
  aliquota = 0.05;
 } else if (valorSalario >= 1100.01 && valorSalario <= 2500.0) {
  aliquota = 0.10;
 } else {
  aliquota = 0.15;
 }
 return aliquota * salario;
}

File: src/logica/Desafio.java
package logica;

import java.util.Scanner;

public class Desafio {
 public static void main(String[] args) {
  try (Scanner leitorDeEntrada = new Scanner(System.in)) {
   System.out.println("Digite seu salário:");
   float valorSalario = leitorDeEntrada.nextFloat();
   System.out.println("Digite o valor do beneficio:");
   float valorBeneficios = leitorDeEntrada.nextFloat();

   float valorImposto = 0;
   if (valorSalario >= 0 && valorSalario <= 1100) {
    valorImposto = 0.05F * valorSalario;
   } else if (valorSalario >= 1100.01 && valorSalario <= 2500.0) {
    valorImposto = 0.10F * valorSalario;
   } else {
    valorImposto = 0.15F * valorSalario;
   }

   float saida = (valorSalario - valorImposto) + valorBeneficios;

   System.out.println(String.format("Valor é %.2f", saida));
  }
 }
}